### PR TITLE
Lonchanick - Calculator console APP 

### PR DESCRIPTION
**Pull request by lonchanick**
Practicing with C# academy

### CODE REVIEW

File: Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            writer.WritePropertyName("Operation");
+            // Use a switch statement to do the math.
+            switch (op)
+            {
+                case "a":
+                    result = num1 + num2;
+                    writer.WriteValue("Add");
+                    break;
+                case "s":
+                    result = num1 - num2;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs:48]: At the moment the result of the operations isn't being printed to the user. Was that your intent?

image

File: Calculator.Lonchanick/Calculator.Lonchanick/Program.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
+﻿using System;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Program.cs:1]: Remove unused usings

File: Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs:5]: Remove unused usings

File: Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Diagnostics;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs:6]: Remove unused usings

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

File: Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs (hunk 4 above)
Comment (lonchanick) [reply Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs]: Fixed

File: Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs (hunk 3 above)
Comment (lonchanick) [reply Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs]: Fixed and new functionalities added

File: Calculator.Lonchanick/Calculator.Lonchanick/Program.cs (hunk 2 above)
Comment (lonchanick) [reply Calculator.Lonchanick/Calculator.Lonchanick/Program.cs]: Fixed and new functionalities added

File: Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs (hunk 1 above)
Comment (lonchanick) [reply Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs:23]: Fixed and new functionalities added

File: Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs [hunk 5]
```diff
@@ -0,0 +1,59 @@
+﻿
+namespace Calculator.Lonchanick
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Calculator.cs:2]: You can use this notation for all your namespaces to save one line of indentation:

image

File: Calculator.Lonchanick/Calculator.Lonchanick/Operations.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public class Operations
+    {
+        public double Operando1 { get; set; }
+        public double Operando2 { get; set; }
+        public double Result{ get; set;}
+        public string Operation { get; set; }
+        public Operations(double op1, double op2, double result, string operation)
+        {
+            this.Operando1 = op1;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Operations.cs:13]: You don't need "this". The compiler takes care of it for you :D 
Usually the compiler will tell you these things when you hover over faded keywords (even though I'll concede you can barely see that "this" is faded here)

image

File: Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs [hunk 7]
```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/ToolBox.cs:1]: Remove unused using

File: Calculator.Lonchanick/Calculator.Lonchanick/Program.cs [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                }
+                
+                calc.DoOperation(num1, num2, op);
+                Console.WriteLine("Press q to close app, or press any other key to continue.. ");
+                if (Console.ReadLine() == "q") flag = true;
+            }
+            calc.CloseLog();
+        }
+
+        static void ShowCurrentOperations()
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Program.cs:50]: Seems like this method isn't being used? Could you please remove it if that's the case.

File: Calculator.Lonchanick/Calculator.Lonchanick/Operations.cs [hunk 9]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
```
Comment (TheCSharpAcademy) [inline Calculator.Lonchanick/Calculator.Lonchanick/Operations.cs:1]: please remove unused using

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]

**Review by lonchanick — COMMENTED** [review]
Amazing!! Thanks!! 
I hand made all the changes, Im a little lost with the Github using but im growing :) !

**Review by TheCSharpAcademy — APPROVED** [review]

**Pull request merged**